packages/syft/src/syft/service/code/user_code.py

```diff
@@ -395,13 +396,16 @@ def debox_asset(arg: Any) -> Any:
 
 def syft_function(
     input_policy: Union[InputPolicy, UID],
-    output_policy: Union[OutputPolicy, UID],
+    output_policy: Union[OutputPolicy, UID] = None,
```

Comment:
Also can we update the typing for `output_policy` to be Optional if we want to set it None by default.